python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -252,6 +288,35 @@ def runDataRef(self, dataRef, selectDataList=None, warpRefList=None):
 
         return results
 
+    def processResults(self, coaddExposure, dataRef):
+        """Interpolate over missing data and mask bright stars.
+
+        Also detect sources on the coadd exposure and measure the final PSF,
+        if ``doCalculatePsf`` is set.
+
+        Parameters
+        ----------
+        coaddExposure : `lsst.afw.image.Exposure`
+            The final coadded exposure.
+        dataRef : `lsst.daf.persistence.ButlerDataRef`
+            Data reference defining the patch for coaddition and the
+            reference Warp
+        """
+        AssembleCoaddTask.processResults(self, coaddExposure, dataRef)
+
+        if self.config.doCalculatePsf:
+            idFactory = self.makeIdFactory(dataRef)
+            expId = int(dataRef.get("dcrCoaddId"))
```

Comment:
Why do you need this `int` cast?

Reply:
I don't think I do need it. I'll remove it.